sdk/textanalytics/azure-ai-textanalytics-perf/README.md

```diff
@@ -0,0 +1,71 @@
+# Azure Text Analytics Performance test client library for Java
+
+Represents Performance tests for Azure Text Analytics SDK for Java.
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
+- [Cognitive Services or Text Analytics account][text_analytics_account] to use this package.
+
+#### Setup for test resources
+
+You will need the following environment variables for running the tests to access the live resources:
+
+```
+AZURE_TEXTANALYTICS_ENDPOINT=<text analytics service endpoint>
+AZURE_TEXTANALYTICS_API_KEY=<text analytics API Key>
+```
+
+### Adding the package to your product
+[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-textanalytics</artifactId>
+    <version>5.1.0-beta.6</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Key concepts
+
+## Examples
+#### Executing the performance test
+1. Compile the performance project into a standalone jar using the command from the root of the perf project folder
+   ```java
+   mvn clean package
+
+2. Execute the corresponding perf test in the project using the command.
+   ```java
+   java -jar <path-to-packaged-jar-with-dependencies-from-step-1> <options-for-the-test>
```

Comment:
can you provide a concrete command example? similar to JS's? It is in the readme here: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/perf-tests/text-analytics/README.md